api/entityapi/actions.py

```diff
@@ -104,6 +104,10 @@ def parse_entry_to_entity(
             {"".join(field_and_val[:1]): ":".join(field_and_val[1:])}
         )
 
+    # limitation of bugout api in update entry
+    created_at = entry.created_at if type(entry) == BugoutJournalEntry else None
```

Comment:
In `BugoutJournalEntry` created_at and updated_at fields could be None, maybe better to use here `try: catch:` block to parse it?

Reply:
Fixed with check of pydentic model __fields__